docs/modules/administration/pages/installation.adoc

```diff
@@ -13,6 +13,16 @@ In order to run the Atlassian suite and OpenShift, your host must have:
 - A recent `git` version (>= 2.13.2)
 - A recent `jq` version (>= 1.5, see https://stedolan.github.io/jq/)
 
+== Service Requirements
+ODS 3.x officially supports:
+- Openshift v3.11.154. The supported images for this version of Openshift can be found at:
+    - https://docs.openshift.com/container-platform/3.11/release_notes/ocp_3_11_release_notes.html
+- Jenkins v2.176.3
+    - The base image of the Jenkins we use through this version of ODS is: `openshift3/jenkins-2-rhel7:v3.11.153-2`
+- Sonarqube 
+    - 7.9 (LTS release)
+    - 8.2.0.32929 (latest release)
```

Comment:
I would just add a comment in the config step to tell people to look at the supported / tested versions when they configure ... and the actual versions are in the sample. I just want to avoid having all sorts of numbers in various places that do not get updated in sync ;)

Reply:
Fair enough, keeps it simpler to maintain